README.md

```diff
@@ -89,6 +89,23 @@ docker-compose -f docker-compose.yml -f docker-compose-elk.yml up -d --build
 docker-compose -f docker-compose.yml -f docker-compose-influx.yml up -d --build
 ```
 
+- [Optional] Install basic management services:
+
+Use the following to install Portainer and Watchtower to manage your single node cluster.
+
+Set the following environment variables before running the services
+
+```
+set VIRTUAL_HOST=portainer.myhost.com
```

Comment:
Could you add these variables in a `.env.management` file to keep the same pattern as the other files